x/staking/legacy/v0_36/types.go

```diff
@@ -29,7 +27,7 @@ type (
 
 	Validator struct {
 		OperatorAddress         sdk.ValAddress          `json:"operator_address" yaml:"operator_address"`
-		ConsPubKey              crypto.PubKey           `json:"consensus_pubkey" yaml:"consensus_pubkey"`
+		ConsPubKey              string                  `json:"consensus_pubkey" yaml:"consensus_pubkey"`
```

Comment:
This should not change. See the correct type: https://github.com/cosmos/cosmos-sdk/blob/v0.36.0/x/staking/types/validator.go#L40